cli/lib/kontena/command.rb

```diff
@@ -218,15 +218,19 @@ def run(arguments)
     end
   rescue Kontena::Errors::StandardError => ex
     raise ex if ENV['DEBUG']
-    abort(" [#{Kontena.pastel.red('error')}] #{ex.class.name} : #{ex.message}")
+    Kontena.logger.error("ERROR") { "#{ex.class.name} : #{ex.message}" }
```

Comment:
Using `"ERROR"` as the `progname` looks weird, it repeats the `ERROR` twice, as the severity and the progname:

`logger.error("ERROR") { "foobar" }` -> `E, [2017-05-04T17:17:33.387675 #10387] ERROR -- ERROR: foobar`